app/desktop/uwp_helper/build.rs

```diff
@@ -0,0 +1,35 @@
+extern crate cbindgen;
+
+use std::env;
+use std::path::PathBuf;
+
+use cbindgen::{Config, Language};
+
+fn main() {
+    let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+
+    let package_name = env::var("CARGO_PKG_NAME").unwrap();
+    let output_file = target_dir()
+        .join(format!("{}.h", package_name))
+        .display()
+        .to_string();
+
+    let mut config = Config::default();
```

Comment:
Why was this changed from the previous syntax?

Reply:
i updated cbindgen and there is a new field that isn't public, so the struct can no longer be initialized manually. also `Hstring.h` is needed for jextract to correctly understand the header.

Reply:
ok i see